.gitignore

```diff
@@ -1,30 +1,5 @@
 /node_modules
-/LBRY-darwin-x64
 /dist
-
-/src/main/dist
-/src/main/locales
-/src/main/node_modules
-/src/renderer/dist
-/build/venv
 /build/daemon.ver
-/lbry-app-venv
-/lbry-app
-/lbry-venv
 /static/daemon/lbrynet*
-/static/locales
-/daemon/build
-/daemon/venv
-/daemon/requirements.txt
-/.idea
-
-*.pyc
```

Comment:
this has to be here while upload_assets.py is still in use. if you want to rewrite that in JS instead, then we can drop python dependency completely, which would be sweet!